src/api/todos.ts

```diff
@@ -0,0 +1,24 @@
+import { TodoType } from '../types/TodoType';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<TodoType[]>(`/todos?userId=${userId}`);
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const addTodo = ({ userId, title, completed }: Omit<TodoType, 'id'>) => {
+  return client.post<TodoType>('/todos', { userId, title, completed });
+};
+
+export const toggleTodoCompleteState = (todoId: number, completed: boolean) => {
+  return client.patch<TodoType>(`/todos/${todoId}`, { completed });
+};
+
+export const updateTodoTitle = (todoId: number, title: string) => {
```

Comment:
you don't need to create separate methods for this, use one

Reply:
@denys-danyliuk but it takes different props i dont know how to make it with one

Reply:
if you have questions you should as them in fe_chat instead of requesting review again, don't pass separate properties, just pass data object where this properties may be